### PR TITLE
refactor :  send messages split into internal and external

### PR DESCRIPTION
**Pull request by wzh425**

### CODE REVIEW

File: src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Mc/Model/SendTemplateMessageByInternalModel.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    public string Sign { get; set; } = string.Empty;
+
+    public List<InternalReceiverModel> Receivers { get; set; } = new();
+
+    public SendRuleModel SendRules { get; set; } = new();
+
+    public ExtraPropertyDictionary Variables { get; set; } = new();
+
+    public Guid OperatorId { get; set; } = default;
```
Comment (zhenlei520) [inline src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Mc/Model/SendTemplateMessageByInternalModel.cs:24]: Use 'Guid.NewGuid()' or 'Guid.Empty' or add arguments to this GUID instantiation

File: src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Mc/Service/IChannelService.cs [hunk 2]
```diff
@@ -8,4 +8,6 @@ public interface IChannelService
     Task<ChannelModel?> GetAsync(Guid id);
 
     Task<PaginatedListModel<ChannelModel>> GetListAsync(GetChannelModel options);
+
+    Task CreateAsync(ChannelUpsertModel options);
```
Comment (zhenlei520) [inline src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Mc/Service/IChannelService.cs:12]: UpsertAsync?

**Review by zhenlei520 — CHANGES_REQUESTED** [review]

File: src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Mc/Service/IChannelService.cs (hunk 2 above)
Comment (wzh425) [reply src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Mc/Service/IChannelService.cs:12]: Allow other projects to create or modify their channels

**Review by zhenlei520 — APPROVED** [review]
LGTM @doddgu

File: src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Mc/Model/ChannelCreateModel.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.BuildingBlocks.StackSdks.Mc.Model;
+
+public class ChannelCreateModel
```
Comment (doddgu) [inline src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Mc/Model/ChannelCreateModel.cs:6]: ChannelCreateModel -> CreateChannelModel

**Review by doddgu — CHANGES_REQUESTED** [review]

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**